certification/internal/engine/engine.go

```diff
@@ -82,7 +87,7 @@ func (c *CraneEngine) ExecuteChecks() error {
 	}()
 
 	log.Debug("extracting container filesystem to ", containerFSPath)
-	err = fileutils.Untar(containerFSPath, r)
```

Comment:
HasMinimalVulns check is also using this method to extract the oval file. Not sure if there is any plan to add it back.

Reply:
We can address that when we get there.